tests/pos/i9266.scala

```diff
@@ -1,3 +0,0 @@
-import language.`3.1-migration`
-
-def test = { implicit x: Int => x + x }
```

Comment:
@anatoliykmetyuk We should not remove this test as it does test that there is no compilation failure after the warning is emitted. Please re-enable it.

Reply:
Ok, re-enabled it.